app.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs')
+const fetch = require('node-fetch')
+const colors = require('colors')
+
+console.log("Natalia Olmos / Proyecto MDLINKS /Laboratoria CDMX009")
+
+function getContentString() {
+
+    let index = process.argv.indexOf("--file")
+    if (index < 0) return console.log("You need to use a valid uri flag --file")
+    let uri = process.argv[index + 1]
+    let string = fs.readFileSync(uri, 'utf8')
+    return string
+}
+
+function getLinks(string) {
+
+    const getArray = ("Text: ", string.toString());
+    let regExpression = /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g;
+    return getArray.match(regExpression);
+}
```

Comment:
Definir mejor los parámetros recibidos
```suggestion
function getLinks(file) {

    const getArrayLinks = ("Text: ", string.toString());
    let regExpression = /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g;
    return getArrayLinks.match(regExpression);
}
```